src/components/AddaccountCard.tsx

```
@@ -0,0 +1,136 @@
+import { useState } from "react";
+
+const AddaccountCard = () => {
+  const [activeState, setActiveState] = useState("crypto");
+  const [step, setStep] = useState(1);
+
+  const handleStep = (e: React.FormEvent) => {
```

ESLint

[error] 'handleStep' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)